### ISSUE TITLE
[A11Y] Seven/Basis: secondary submit button text does not have enough contrast

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#5426]
This is a sibling issue to #5425 and part of the #5244 meta.

<img width=50% src="https://user-images.githubusercontent.com/2423362/147845222-b85dd9f4-2634-4899-9619-8762b0c495e9.png">

Current style is `#828188` text color over `#FFFFFF` background, which is AA for large text, but fails for normal size text.

Making the text color `#75747B` would pass AA for both Seven/Basis (would make it AAA for large size text, and AA for normal size text).

Making it `#57575C` would pass AAA for both Seven/Basis, for both large and normal size text.

PS: we need to also:
- match the button border color with the text color, and then make sure that that has enough contrast against the page background
- check contrast of these buttons on hover as well

**Comment by klonos** [comment]
PRs filed for both options to compare and choose from:
- AA: https://github.com/backdrop/backdrop/pull/3885
- AAA: https://github.com/backdrop/backdrop/pull/3886

**Comment by klonos** [comment]
Although AAA is ideal, we don't want to draw attention to secondary buttons, so perhaps we can call AA "good enough" and choose that here.

**Comment by indigoxela** [comment]
> Although AAA is ideal, I believe that we don't want to draw attention to secondary buttons

Agreed, for the same reason. Another vote for the AA approach for both, Seven and Basis.

**Comment by BWPanda** [comment]
I also think AA is good enough. The AAA ones look too bold...

PR looks good to me.

**Comment by quicksketch** [comment]
> PS: I have not changed the border (box-shadow) for these buttons, as I think it makes for a nice "bezel" effect. Happy to change that too, to match the button text or background color (depending on normal vs hover/focus state).

I think we should change the border to match the background, as these buttons did not have a different color border before. I also find it weird that the secondary buttons would get a visible border while the primary button would not.

So while we agree going with AA is good enough, I think this needs an update to change the border to match.
